src/Renderer/PointsMaterial.js

```diff
@@ -28,7 +76,22 @@ class PointsMaterial extends THREE.RawShaderMaterial {
         CommonMaterial.setUniformProperty(this, 'picking', false);
         CommonMaterial.setUniformProperty(this, 'opacity', this.opacity);
         CommonMaterial.setUniformProperty(this, 'overlayColor', options.overlayColor || new THREE.Vector4(0, 0, 0, 0));
-        CommonMaterial.setUniformProperty(this, 'intensityRange', options.intensityRange || new THREE.Vector2(0, 1));
+        CommonMaterial.setUniformProperty(this, 'intensityRange', intensityRange);
+
+        // add classification texture to apply classification lut.
+        const [width, height] = [256, 1];
+        const data = new Uint8Array(width * 4);
+        const texture = new THREE.DataTexture(data, width, height, THREE.RGBAFormat);
+        texture.magFilter = THREE.NearestFilter;
+
+        this.classificationTexture = texture;
```

Comment:
This can be optional, no ? You can use `texture` directly, and use `this.classificationLUT` instead of this.

Reply:
yes